src/style.css

```diff
@@ -0,0 +1,26 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
No.. no.. no, don't set
margin: 0;
padding: 0;
to all elements.. u set this prop to an element which never has margins